content/integrations/statsd.md

```diff
@@ -12,7 +12,9 @@ Enable the Datadog StatsD integration to:
   * Monitor uptime and availability of a StatsD server
   * Collecting metrics from a StatsD server's admin interface
 
-**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. This integration is intended for monitoring external StatsD servers, and is not needed to send metrics to Datadog using the StatsD protocol.**
+**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. If you want to send metrics from a statsd server, just point it to the forwarder listening on port 17123.**
```

Comment:
> the problem is that folks install the integration thinking that they need to to get metrics from a 3rd party statsd agent thingy.

I don't follow...

Whatever the confusion is, linking them to the DogStatsD page doesn't help, as that page doesn't describe how to forward StatsD metrics to Datadog. Even if it did describe it, the sentence is still confusing:

> Please note the Datadog Agent includes DogStatsD, which serves as a StatsD forwarder.

Readers who don't follow the link may think: "Great, yes, that's what I want to do! Sounds like I'm all set." The sentence makes it sound like StatsD forwarding is working out of the box as long as dd-agent is installed.